doc/conf.py

```diff
@@ -3,42 +3,65 @@
 import sys, os
 sys.path.insert(0, os.getcwd())
 
-from builder.shared_conf import * # noqa (API import)
-
-paths = ['../param/', '.', '..']
-add_paths(paths)
-
-from ..setup import setup_args
+from nbsite.shared_conf import *
 
 # Declare information specific to this project.
 project = u'HoloViews'
 authors = u'PyViz developers'
-copyright = u'2018 ' + authors
-ioam_module = 'holoviews'
+copyright = u'2019 ' + authors
+pyviz_module = 'holoviews'
 description = 'Stop plotting your data - annotate your data and let it visualize itself.'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = setup_args['version']
+version = 'v1.11.0' #setup_args['version']
```

Comment:
Is the version not accessible?

Reply:
I had trouble with it locally but will try to restore it before merging.

Reply:
This is what we do on our autover projects 

```python
import colorcet
version = release = colorcet.__version__
```